### DISCUSSION TITLE
ZOOMEYE_API_KEY env variable exists but ZOOMEYE_HOST does not ERROR

### DISCUSSION THREAD

**Discussion by emirhanmtl** [projectdiscovery/nuclei#5375] [Q&A]
Hello, I am getting "[ERR] ZOOMEYE_API_KEY env variable exists but ZOOMEYE_HOST does not" this error when I try to start nuclei. How can I fix this can someone help?

**Comment by liushastil** [comment]
There is an answer in the following link.
https://github.com/projectdiscovery/uncover/pull/535/files

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
